New resource/data source: `azurerm_mssql_database` (#6083)
terraform-providers/terraform-provider-azurerm: website/docs/d/mssql_database.html.markdown

---
subcategory: "Database"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_mssql_database"
description: |-
  Gets information about an existing SQL database.
---

# Data Source: azurerm_mssql_database

Use this data source to access information about an existing SQL database.

## Example Usage

```hcl
data "azurerm_mssql_database" "example" {
  name          = "example-mssql-db"
  sql_server_id = "example-mssql-server-id"
}

output "database_id" {
  value = data.azurerm_mssql_database.example.id
}
```

## Argument Reference

* `name` - The name of the Ms SQL Database.

* `sql_server_id` - The id of the Ms SQL Server on which to create the database.

> ## Argument Reference

Reviewer: ```suggestion
## Attribute Reference
```

* `collation` - The collation of the database. 

> * `elastic_pool_id` - The id of the elastic pool containing this database.

Reviewer 2: ```suggestion
* `elastic_pool_id` - The ID of the elastic pool containing this database.
```

> * `license_type` - The license type to apply for this database.

Reviewer 2: ```suggestion
* `license_type` - The license type applied to this database.
```

* `max_size_gb` - The max size of the database in gigabytes.

* `read_replica_count` - The number of readonly secondary replicas associated with the database to which readonly application intent connections may be routed. 

* `read_scale` - If enabled, connections that have application intent set to readonly in their connection string may be routed to a readonly secondary replica.

> * `sku_name` - The name of the sku of the database.

Reviewer 2: ```suggestion
* `sku_name` - The name of the sku used by the database.
```
